components/Headline.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export default function Headline({ name }) {
+  return (
+    <StyledHeadline>
+      <StyledH1>{name}</StyledH1>
+    </StyledHeadline>
+  );
+}
+
+const StyledHeadline = styled.div`
+  margin: 0 var(--basicmargin);
+`;
+const StyledH1 = styled.div`
```

Comment:
```suggestion
const StyledH1 = styled.h1`
```
I take from your naming that this should be an "h1" or? Nevertheless, I would recommend you a more descriptive naming here, e.g. like `StyledHeadline` and the wrapping `div` could be named `StyledHeadlineWrapper`.